aws/aws.go

```diff
@@ -177,8 +191,21 @@ func NukeAllResources(account *AwsAccountResources, regions []string) error {
 
 		resourcesInRegion := account.Resources[region]
 		for _, resources := range resourcesInRegion.Resources {
-			if err := resources.Nuke(session); err != nil {
-				return errors.WithStackTrace(err)
+			if len(resources.ResourceIdentifiers()) >= 500 {
+				logging.Logger.Infoln("Terminating Resources in batches")
+				batches := split(resources.ResourceIdentifiers(), 200)
```

Comment:
Where does `500` and `200` come from? These are "magic" numbers and should be extracted into constants with some explanation above them (e.g., AWS throttles after XXX API calls...).

Reply:
In fact, each resource type should probably have a `MaxBatchSize()` function.

Reply:
Sorry about the "magic" numbers. Was trying stuff out and forgot to abstract it.

Reply:
AWS is vague on what the limits are: https://docs.aws.amazon.com/AWSEC2/latest/APIReference/query-api-troubleshooting.html. However, they vary per service and even per region. See also https://stackoverflow.com/questions/23728591/ec2-api-throttling-limits. 

I suspect we will need the flexibility to use different limits with different services. Also, it may turn out simple batching doesn't work and we need an [exponential back off](https://docs.aws.amazon.com/general/latest/gr/api-retries.html), but let's try the simple version first.

Reply:
Ok, Gotcha. Thanks for the links